lightning/src/ln/channelmanager.rs

```diff
@@ -3474,12 +3484,12 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 			for chan in self.list_channels() {
 				if chan.remote_network_id == *counterparty_node_id {
```

Comment:
Why you don't call `force_close..` with Some(*counterparty_node_id)?

Reply:
Doesn't matter much, but, sure, probably better for once we index by peers anyway.